### DISCUSSION TITLE
How to loki update for android?

### DISCUSSION THREAD

**Discussion by lpatrun** [oblador/loki#343] [Q&A]
Recently I am having no success with trying to take screenshots from android emulator in my projects so I decided to do a side step and try with example found in the repo.

Step 1: download and unzip the example react-native
Step 2: Install packages, I installed them with yarn
Step 3: yarn storybook
Step 4: react-native run-android
Step 5: yarn loki update

In the attached ss you can clearly see that I am able to start the storybook, but I get this weird error when trying to make the screenshots.

Anybody having success with this?

<img width="1427" alt="Screenshot 2021-10-23 at 22 34 30" src="https://user-images.githubusercontent.com/11506643/138570700-135c3a0f-f8a2-4233-a2a4-156f0fa85dc3.png">

**Comment by cassiourugit** [comment]
I have the same issue. Works fine with iOS simulator. Only android emulator does not work

**Reply by andreyprok** [reply]
same issue, Ios emulator does-not work as well, storybook8, any workarounds?
